src/static/helpers/slasHelper.test.ts

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as slasHelper from './slasHelper';
+import {ShopperLogin} from '../../lib/shopperLogin';
+
+const authenticateCustomerMock = jest.fn(options => ({
+  url: 'https://localhost:3000/callback?usid=048adcfb-aa93-4978-be9e-09cb569fdcb9&code=J2lHm0cgXmnXpwDhjhLoyLJBoUAlBfxDY-AhjqGMC-o',
+}));
+
+const authorizeCustomerMock = jest.fn(options => ({
+  url: 'https://localhost:3000/callback?usid=048adcfb-aa93-4978-be9e-09cb569fdcb9&code=J2lHm0cgXmnXpwDhjhLoyLJBoUAlBfxDY-AhjqGMC-o',
+}));
+
+const getAccessTokenMock = jest.fn(options => ({access_token: 'access_token'})); // TODO: replace with proper access token
```

Comment:
Do we need to create stories for TODO's?

Reply:
Nope, sorry those are just reminders for me. I wanted to push the unit tests I had already so I could get feed back as soon as I could